iam/api-client/workload_identity_federation.py

```diff
@@ -1,4 +1,4 @@
-#  Copyright 2021 Google LLC
+#  Copyright 2022 Google LLC
```

Comment:
The license year does not need to be updated, it should reflect the year that the file was first introduced.

Reply:
Thanks for the information. Resolved.